templates/travel/partials/hero/_hero.css

```diff
@@ -0,0 +1,229 @@
+
+.travel-hero {
+  height: 100vmax;
+}
+
+.travel-hero-bg {
+  height: 70vmax;
+  animation: fade 0.8s ease-out both;
+}
+
+.travel-hero-bg-img {
+  top: -5vmax;
+  right: -5vmax;
+  bottom: -5vmax;
+  left: -5vmax;
+  opacity: 0;
+  background: var(--silver)
+}
+.travel-hero-bg-img img {
+  object-fit: cover;
+}
+.travel-hero-bg-img:nth-child(3) img {
+  object-position: 80% 100%;
+}
+
+.travel-hero-bg-img:nth-child(1) {
+  animation: 18s travel-hero-scene1 linear infinite;
+  background: url('../../img/travel/hero-2-preview.jpg') no-repeat 50% 50%;
+  background-size: cover;
+}
+
+.travel-hero-bg-img:nth-child(2) {
+  animation: 18s travel-hero-scene2 linear 6s infinite;
+  background: url('../../img/travel/hero-3-preview.jpg') no-repeat 50% 50%;
+  background-size: cover;
+}
+
+.travel-hero-bg-img:nth-child(3) {
+  animation: 18s travel-hero-scene3 linear 12s infinite;
+  background: url('../../img/travel/hero-1-preview.jpg') no-repeat 50% 50%;
+  background-size: cover;
+}
+
+.travel-hero-bg-fade {
+  height: 70vmax;
+  top: calc(70vmax - (300 / 16rem));
+  background-image: linear-gradient(180deg, transparent 0%, var(--black) calc(300 / 16rem));
+}
+
+.travel-hero-content {
+  padding-top: var(--fixed-navbar-height);
+  height: calc(100vmax - 70px);
+}
+
+.travel-hero-heading {
+  color: var(--white);
+  font-size: calc(60 / 16rem);
+  font-weight: 700;
+}
+
+.travel-hero-subheading {
+  color: var(--white);
+  font-size: calc(18 / 16rem);
+}
+
+.travel-hero-search-dates {
+  color: var(--white);
+}
+
+.travel-hero-heading {
+  animation: fade-and-slideup 0.4s 0.2s ease-in-out both;
+}
+
+.travel-hero-subheading {
+  animation: fade-and-slideup 0.4s 0.25s ease-in-out both;
+}
+
+.travel-hero-search .input-icon {
+  animation: fade-and-slideup 0.4s 0.3s ease-in-out both;
+}
+
+.travel-hero-search-dates {
+  animation: fade-and-slideup 0.4s 0.35s ease-in-out both;
+}
+
+.travel-hero-search .ampstart-btn {
+  font-size: 1.25rem;
+  background-color: var(--purple);
+  animation: fade-and-slideup 0.4s 0.4s ease-in-out both;
+}
+
+.travel-hero-discover {
+  color: var(--gray);
+  animation: fade-and-slideup 0.4s 0.45s ease-in-out both;
+}
+
+.travel-angles {
+  animation: fade-and-slideup 0.4s 0.5s ease-in-out both;
+}
+
+.travel-discover {
+  animation: fade-and-slideup 0.4s 0.55s ease-in-out both;
+}
+
+.travel-activities {
+  animation: fade-and-slideup 0.4s 0.6s ease-in-out both;
+}
+
+@media (--breakpoint-md) {
+  .travel-hero {
+    z-index: 1;
+    height: auto;
+  }
+
+  .travel-hero-bg {
+    position: relative;
+    height: auto;
+  }
+
+  .travel-hero-content {
+    padding-top: calc(var(--fixed-navbar-height) + (56 / 16rem));
+    padding-bottom: calc(48 / 16rem);
+    height: auto;
+    position: relative;
+  }
+
+  .travel-hero-heading {
+    font-size: var(--h00);
+  }
+
+  .travel-hero-subheading {
+    font-size: calc(24 / 16rem);
+  }
+
+  .travel-hero-search {
+    margin-top: var(--space-5);
+    max-width: calc(530 / 16rem);
```

Comment:
still dont understand what the 530 and 16 are ? you need to add comments or use variables to explain what is going on here

Reply:
16 is 16px. (530px / 16px * 1rem). I'll add comments.

Reply:
i mean where does the 530 come from and where does the 16 come from. 

Also there are max-width-1,2,3 utils in bass

Reply:
530px is the max-width we want the hero text to be (for design reasons). 16px is the base font-size that helps us convert px to rems. (530px = 33.125rem, which is narrower than the variables widths used in the basscss max-width utils)

Reply:
Why cant we just say max-width: 530px?

Reply:
done - 74d7f18